_posts/2013-01-01-tcp_protocol_spec.md

```diff
@@ -290,6 +290,9 @@ Similarly, a message that is in-flight and times out behaves identically to an e
     <timeout> - a string representation of integer N where N <= configured max timeout
         0 is a special case that will not defer re-queueing
 
+        timeout == 0 - requeue a message immediately
+        timeout  > 0 - asynchronously wait for the specified timeout and requeue a message ("deferred queueing")
```

Comment:
I still don't think this is the best wording ... I think it's a bit more important to mention that timeout is in _milliseconds_ ... other than that I think it would be sufficient to just say that the re-queue is deferred for timeout milliseconds.